README.md

```diff
@@ -33,6 +33,7 @@ Note that queries are PostgreSQL-specific.
 - `bundle exec guard` - Runs the guard test server that reruns your tests after files are saved. Useful for TDD!
 - `bundle exec rake security` - Run the suite of security scanners on the codebase.
 - `bundle exec rake ci` - Run all build steps performed in Travis CI.
```

Comment:
Noticed this.  Can you remove "Travis" since we do CI ourselves (jenkins) now?

Reply:
updated description